src/govuk/components/notification-banner/notification-banner.test.js

```diff
@@ -21,6 +21,15 @@ describe('/components/notification-banner/with-type-as-success', () => {
 
     expect(activeElement).toBe('govuk-notification-banner')
   })
+
+  it('removes the tabindex attribute on blur', async () => {
+    await page.goto(baseUrl + '/components/notification-banner/with-type-as-success/preview', { waitUntil: 'load' })
+
+    await page.$eval('.govuk-notification-banner', el => el.blur())
+
+    const tabindex = await page.$eval('.govuk-notification-banner', el => el.getAttribute('tabindex'))
+    expect(tabindex).toBeFalsy()
```

Comment:
I think we could probably use `toBeUndefined ` here – `toBeFalsy` wouldn't fail if `tabindex` was 0, for example.

```suggestion
    expect(tabindex).toBeUndefined()
```

Reply:
`toBeFalsy()` does fail here if `tabindex` is set to 0. 

Puppeteer returns `null` for things that don't exist in the DOM so for this reason `toBeUndefined()` can't used here. 

I think we're okay to leave this as is but let me know if you think further digging is needed 🙏

Reply:
Ah, I guess because it's the string "0" rather the number 0?

I think [`toBeNull`](https://jestjs.io/docs/en/expect#tobenull) might be what we want, in that case?

```suggestion
    expect(tabindex).toBeNull()
```